eslint.config.js

```diff
@@ -0,0 +1,444 @@
+const { FlatCompat } = require("@eslint/eslintrc");
```

Comment:
You can write all these imports with the import syntax to modernize it too, like so:
```js
import { FlatCompat } from'@eslint/eslintrc';
import pluginImport from 'eslint-plugin-import';
```
etc.

Reply:
yes, would make it eslint.config.mjs and add the changes